cmd/experimental/kjobctl/pkg/builder/builder.go

```diff
@@ -588,6 +591,19 @@ func (b *Builder) withKueueLabels(objectMeta *metav1.ObjectMeta) {
 	if len(b.priority) != 0 {
 		objectMeta.Labels[kueueconstants.WorkloadPriorityClassLabel] = b.priority
 	}
+
+	if b.timeLimit != "" {
+		maxExecutionTimeSeconds, err := parser.TimeLimitToSeconds(b.timeLimit)
+		if err != nil {
+			return fmt.Errorf("cannot parse '%s': %w", b.timeLimit, err)
+		}
+
+		if maxExecutionTimeSeconds != nil {
```

Comment:
Maybe 
```suggestion
		if ptr.Deref(maxExecutionTimeSeconds, 0) > 0 {
```
if we want to treat `0` as `no limit`.